link/syscalls.go

```diff
@@ -134,3 +134,17 @@ func bpfIterCreate(attr *bpfIterCreateAttr) (*internal.FD, error) {
 	}
 	return nil, err
 }
+
+type bpfRawTracepointOpenAttr struct {
+	name internal.Pointer
+	fd   uint32
+	pad  uint32
```

Comment:
`pad` is not part of the `BPF_RAW_TRACEPOINT` `bpf_attr`. Can we use `_` instead?

Reply:
Oh, sure. my bad :)